db/migrations/20230106041047_create_sub_categories.sql

```diff
@@ -0,0 +1,8 @@
+-- migrate:up
+CREATE TABLE sub_categories (
+id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+country VARCHAR(100) NOT NULL,
+content VARCHAR(200) NOT NULL,
```

Comment:
SQL 포맷팅을 맞춰서 작성해주세요!
```suggestion
CREATE TABLE sub_categories (
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    country VARCHAR(100) NOT NULL,
    content VARCHAR(200) NOT NULL,
```